packages/colorpickers/src/styled/ColorDialog/StyledButton.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Button } from '@zendeskgarden/react-buttons';
+
+const COMPONENT_ID = 'colorpickers.colordialog.button';
+
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
+  const shade = 600;
+  const baseColor = getColor('neutralHue', shade, props.theme);
+  const hoverColor = getColor('neutralHue', shade + 100, props.theme);
+  const activeColor = getColor('neutralHue', shade + 200, props.theme);
+  const disabledForegroundColor = getColor('neutralHue', shade - 200, props.theme);
+
+  return `
+    color: ${baseColor};
+
+    &:hover {
+      color: ${hoverColor};
+    }
+
+    &:active,
+    &[aria-pressed='true'],
+    &[aria-pressed='mixed'] {
+      color: ${activeColor};
+    }
+
+    &:disabled {
+      color: ${disabledForegroundColor};
+    }
+  `;
+};
+
+export const StyledButton = styled(Button).attrs({
```

Comment:
In addition to the items @steue mentioned, there seem to be some styling problems with this component. 

1. Chevron-side padding overflows the button
2. The component isn't rendering right in IE11

<img width="90" alt="Screen Shot 2021-02-17 at 5 26 01 PM" src="https://user-images.githubusercontent.com/143773/108290439-41780980-7145-11eb-9066-16da6ef93bf7.png">